ogre2/include/ignition/rendering/ogre2/Ogre2ArrowVisual.hh

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2020 Open Source Robotics Foundation
+ * Copyright (C) 2022 Open Source Robotics Foundation
```

Comment:
how about keeping the original copyright year? I think https://github.com/gazebosim/gz-rendering/pull/736 backported this class to citadel and changed to date to 2022. So we can just keep the original year when forward porting?

Reply:
I had the same issue in `common`, though I don't think I fixed the copyright years...